src/components/member-card/index.js

```diff
@@ -45,7 +44,7 @@ const Card = ({ developerInfo, isOptionKey }) => {
       {dev && (
         <SuperUserOptions username={username} showSettings={showSettings} />
       )}
-      <motion.img
+      <img
```

Comment:
why?

Reply:
because otherwise, the images of members were sliding down.